content/en/docs/overview/dataplane-modes/index.md

```diff
@@ -276,19 +276,11 @@ The overhead for processing protocols at Layer 7 is substantially higher than pr
   </tbody>
 </table>
 
-## Supported features
+## Unsupported features
 
-As of Istio 1.22, the following ambient mode features are implemented but are currently in Alpha status:
+The following features are available in sidecar mode, but not yet implemented in ambient mode:
```

Comment:
We do have some support for it.  It actually worked as long as there is no waypoint on destination side.  And I know some of our users use it. I'd recommend to reword this.